deploy/chart/values.yaml

```
@@ -19,6 +19,8 @@ writeStatusName: '""'
 imagestream: false
 debug: false
 installType: upstream
+catalogGrpcServicePort: 50051
```

It may be that the service port and pod port are the same, but we specifically need the pod port, so let's change this to be clear that the policy is about the pod port, not the service port.

```suggestion
catalogGrpcPodPort: 50051
```